profiling/simple-kernel-timer/Makefile

```diff
@@ -1,12 +1,12 @@
 CXX=g++
-CXXFLAGS=-O3 -std=c++11 -g
+CXXFLAGS=-O3 -std=c++17 -g
 SHARED_CXXFLAGS=-shared -fPIC
 
 all: kp_kernel_timer.so kp_reader kp_json_writer
 
 MAKEFILE_PATH := $(subst Makefile,,$(abspath $(lastword $(MAKEFILE_LIST))))
 
-CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all
+CXXFLAGS+=-I${MAKEFILE_PATH} -I${MAKEFILE_PATH}/../../common/makefile-only -I${MAKEFILE_PATH}../all -I${MAKEFILE_PATH}../../common
```

Comment:
This is good for now. A minor comment is that I wonder if this could be abstracted out into a make.def file to be accessed by any Kokkos Tools utility.